recipes/diligent-core/all/conanfile.py

```diff
@@ -1,9 +1,13 @@
-from conans import ConanFile, tools, CMake
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conans import CMake
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import cross_building, check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.files import rm, get, rmdir, rename, collect_libs, patches
+from conan.tools.apple import is_apple_os
 import os
-from conan.tools.build import cross_building
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.50.0"
```

Comment:
```suggestion
required_conan_version = ">=1.51.3"
```
is_apple_os: https://docs.conan.io/en/latest/changelog.html#id30